rest/client/rest.go

```diff
@@ -214,3 +214,26 @@ func (c *communicatorImpl) RestartRecentTasks(ctx context.Context, startAt, endA
 
 	return nil
 }
+
+func (c *communicatorImpl) GetDistributionsList(ctx context.Context) (*[]model.APIDistro, error) {
```

Comment:
- don't make pointers to slices. 
- call it "distros"... we should be consistent what we call the distros construct internally, and it's not really a "distro" in a sense that is used anywhere else.

Reply:
I put it there after talking with Brian.

How does `evergreen host list --spawnable` sound for this functionality?

Reply:
I thought about that but I think having it in the host subcommand will be confusing since it sounds like you want to list spawned hosts (which already is a different command)

Reply:
the "evergreen host" sub-command is only for spawn host management, so I feel like you wouldn't need the extra argument," but seems legt, feel free to disregard this.